cmd/api-headers.go

```diff
@@ -47,7 +47,9 @@ func setEventStreamHeaders(w http.ResponseWriter) {
 
 // Write http common headers
 func setCommonHeaders(w http.ResponseWriter) {
-	w.Header().Set(xhttp.ServerInfo, "MinIO/"+ReleaseTag)
+	// Set the "Server" http header.
+	w.Header().Set(xhttp.ServerInfo, "MinIO")
```

Comment:
Maybe 'MinIO server' ? :1st_place_medal:

Reply:
It's like nginx it doesn't say `nginx server`